scrapers/brandibelle.yml

```diff
@@ -0,0 +1,23 @@
+name: brandibelle
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - new.brandibelle.com/video
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="vd-hdd"]/text()
+      Details:
+        selector: //div[@class="vd-desc"]/text()
+        concat: "<br>"
```

Comment:
Was there a scene where you needed the concat or that a `<br> ` was present in the details ?
This could either be 
```yaml
      Details:
        selector: //div[@class="vd-desc"]
```
or 
```yaml
      Details:
        selector: //div[@class="vd-desc"]/text()
        concat: "\n"
```

Reply:
thought there were new lines. But apparently they weren't.